ssts/segmentation/example.py

```diff
@@ -228,6 +229,10 @@ def main(data_path=None, example_number=None):
         # Plot grain classification
         mm.show_classification(seg2_labels, no_outliers_data)
 
+    # NOTE A-priori correlation analysis
+    elif example_number == "8": # NOTE this looks better the more data files in the dir
```

Comment:
Example shows how to get correlation information across data files (inside the same directory)